api/cluster/templater.go

```diff
@@ -208,6 +252,21 @@ func createPredictorSpec(modelService *models.Service, config *config.Deployment
 					Container: corev1.Container{
 						Name:      kserveconstant.InferenceServiceContainerName,
 						Resources: resources,
+						LivenessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								HTTPGet: &corev1.HTTPGetAction{
+									Path:   fmt.Sprintf("/v1/models/%s", modelService.Name),
+									Scheme: "HTTP",
+									Port: intstr.IntOrString{
+										IntVal: 8080,
+									},
+								},
+							},
+							InitialDelaySeconds: 10,
```

Comment:
How do ypu drcide these values?

Reply:
As discussed with @ariefrahmansyah I don't observe any delay in model starting. It is quite instant. Hence a 10s initial delay.

Reply:
Got, do you know what is the impact to slow model that are not able to answer after 10s delay?

Reply:
Do you suggest adding a readiness probe then?

Reply:
I think adding readiness probe and make it same as liveness probe is good idea, wdyt @ariefrahmansyah . Another thing is that can we make these values configurable at merlin level? Additionally we probably should provide configuration to disable it in case there is issue with specific user's deployment.

Reply:
I think liveness probe should be less sensitive than readiness probe. So if the model is not ready yet (e.g. downloading huge size of model), we still give it a chance to finish its job and be ready. If the model 

+1 on making it configurable.